Libraries/PeriphDrivers/Source/LP/lp_me14.c

```diff
@@ -574,3 +549,226 @@ void MXC_LP_ICacheXIPLightSleepEnable(void)
 {
     MXC_GCR->memckcn |= MXC_F_GCR_MEMCKCN_ICACHEXIPLS;
 }
+
+/*
+ *  Switch the system clock to the HIRC / 4
+ *
+ *  Enable the HIRC, set the divide ration to /4, and disable the 96 MHz oscillator.
+ */
+void switchToHIRCD4(void)
```

Comment:
Should this be a static function as well?

Reply:
Yes, the more static functions the better.